internal/imports/imports_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package imports
+
+import (
+	"fmt"
+	"go/token"
+	"reflect"
+	"testing"
+)
+
+var blacklist = []string{
```

Comment:
We should either test this on std or add a non-"testdata" testdata directory.

Reply:
sorry, I don't understand.
why hit the real filesystem when all we need is to test the `go/types` API?